test/verifiedTests/testGDLS/testGDLS.m

```diff
@@ -0,0 +1,36 @@
+function statusOK = testGDLS()

```

Comment:
Remove commits 9c6a07eab1df9997840ce70ef15a65edc2e74eca and 421625e00b8aae5b3fa8501f5eb5e0ef9d085857 from this PR.